katas/metal-slug/jeanpeare/Character.js

```diff
@@ -0,0 +1,61 @@
+export class Character {
+    constructor (CharacterType, Health) {
+        this.CharacterType = CharacterType;
+        this.Health = Health;
+    }
+}
+
+export const Civilian = new Character (
+    'Civilian',
+    1);
+
+export class Player extends Character{
+    constructor(CharacterType, Health, Life){
```

Comment:
for parameters/local variables, please use camel case, this will help to prevent confusion with classes usage